toolset/setup/linux/languages/php7.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+RETCODE=$(fw_exists ${IROOT}/php.installed)
+[ ! "$RETCODE" == 0 ] || { \
+  echo "Moving PHP config files into place";
+  sudo cp $FWROOT/config/php.ini /usr/local/lib/php.ini
+  sudo cp $FWROOT/config/php-fpm.conf /usr/local/lib/php-fpm.conf
```

Comment:
However, once we see that php needs to be globally installed, we find the real problem; going back and forth from php5 to php7 could very well break based on this config files having different layouts.

Maybe I'm being pessimistic and this will just work.